@navikt/core/css/form/search.css

```diff
@@ -0,0 +1,112 @@
+:root {
+  --navds-search-color-border: var(--navds-semantic-color-border);
+  --navds-search-color-border-hover: var(
+    --navds-semantic-color-interaction-primary
+  );
+}
+
+.navds-search--relative-flex {
+  display: flex;
+  position: relative;
+  align-items: center;
+}
+
+.navds-search__input {
+  padding-right: var(--navds-spacing-12);
+  border-top-right-radius: 0;
+  border-bottom-right-radius: 0;
+}
+
+.navds-form-field--small .navds-search__input {
+  min-height: 32px;
+  padding: 0.15rem;
+  padding-right: var(--navds-spacing-8);
+}
+
+/* Clear Button */
+.navds-search__button-clear {
+  position: absolute;
+  right: 0.75rem;
+  border-radius: 4px;
+  color: var(--navds-semantic-color-text);
+  height: 32px;
+  width: 32px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  cursor: pointer;
+  background: none;
+  border: none;
+  font-size: 1rem;
+}
+
+.navds-form-field--small .navds-search__button-clear {
+  right: 0.5rem;
+  height: 24px;
+  width: 24px;
+}
+
+.navds-search__button-clear svg {
+  flex-shrink: 0;
+}
+
```

Comment:
Jeg kan ikke se at det er behov for denne.

```suggestion
```

Reply:
<img width="50" alt="Screenshot 2022-03-01 at 12 19 30" src="https://user-images.githubusercontent.com/26967723/156160071-b2729f92-71e7-4d37-9613-7bf5b3c4bbd2.png">
<img width="48" alt="Screenshot 2022-03-01 at 12 19 34" src="https://user-images.githubusercontent.com/26967723/156160072-e59a3fc5-1424-47d6-ab07-377855e1a409.png">

Blir mindre ute, men kan hende det bare er en sideffekt av noe annet. Skal undersøke.

Reply:
Fixed, var siden padding ikke ble reset og da fikk default button padding.